src/main/java/com/mercadolibre/be_java_hisp_w31_g3/controller/UserController.java

```diff
@@ -0,0 +1,30 @@
+package com.mercadolibre.be_java_hisp_w31_g3.controller;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.mercadolibre.be_java_hisp_w31_g3.dto.FollowersCountDto;
+import com.mercadolibre.be_java_hisp_w31_g3.service.IUserService;
+import com.mercadolibre.be_java_hisp_w31_g3.service.UserService;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+@RestController
+@RequestMapping("/api/users")
```

Comment:
```suggestion
@RequestMapping("/users")
```

El "api" no es necesario